pkg/doc/verifiable/credential_ldp_test.go

```diff
@@ -70,10 +70,22 @@ func TestNewCredentialFromLinkedDataProof_Ed25519Signature2018(t *testing.T) {
 }
 
 //nolint:lll
-func TestNewCredentialFromLinkedDataProof_Ed25519Signature2018_Transmute(t *testing.T) {
+func TestNewCredentialFromLinkedDataProof_JSONLD_Validation(t *testing.T) {
```

Comment:
@kdimak do you have a followup where VPs are being similarly tested?

Reply:
@troyronda In this PR, I did not touch VPs, I will do this in the next PR.